src/support.cpp

```diff
@@ -294,6 +295,27 @@ void AreaSupport::generateSupportAreas(SliceDataStorage& storage, unsigned int m
             supportLayer_this = AreaSupport::join(supportLayer_last, supportLayer_this, join_distance, smoothing_distance, max_smoothing_angle, conical_support, conical_support_offset, conical_smallest_breadth);
         }
 
+        // make towers for small support
+        if (use_towers)
+        {
+            for (ConstPolygonRef poly : supportLayer_this)
+            {
+                if (poly.area() < supportMinAreaSqrt * supportMinAreaSqrt)
+                {
+                    if (layer_idx < support_layer_count - tower_top_layer_count && layer_idx >= 1)
+                    {
+                        const Polygons& support_layer_above = supportAreas[layer_idx + tower_top_layer_count];
+                        Polygons here;
+                        here.add(poly);
+                        if (here.intersection(support_layer_above).size() > 0)
+                        {
+                            towerRoofs.push_back(here);
```

Comment:
This just adds an already-existing support area to the tower roofs? What will that do?

(Not blocking, just asking for clarification.)